assets/stylesheets/_z-index.scss

```diff
@@ -77,7 +77,10 @@ $z-layers: (
 
 	// Shows adminbar quicklink submenu above bottom popover:
 	// #wpadminbar ul li {z-index: 99999;}
-	".components-popover.is-bottom": 99990,
+	".components-popover:not(.is-mobile).is-bottom": 99990,
```

Comment:
This commit breaks visibility of `Dropdown` on mobile when used in `PluginSidebar`. Reverting z-index back to `z-index: 1000000` here fixes the issue.

ping @jasmussen

Reply:
Whoops, perhaps it wasn't THIS commit, but this is the line that breaks `Dropdown`.